src/contracts/core/AVSDirectoryStorage.sol

```diff
@@ -42,5 +55,5 @@ abstract contract AVSDirectoryStorage is IAVSDirectory {
      * variables without shifting down storage in the inheritance chain.
      * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
      */
-    uint256[47] private __gap;
+    uint256[43] private __gap;
```

Comment:
I'm seeing 3 new mappings but we decremented gap by 4??

Reply:
Fixed